packages/contracts-bedrock/snapshots/storageLayout/L1StandardBridge.json

```diff
@@ -1,24 +1,10 @@
 [
   {
-    "bytes": "1",
-    "label": "_initialized",
+    "bytes": "32",
+    "label": "spacer_0_0_32",
     "offset": 0,
     "slot": "0",
-    "type": "uint8"
-  },
-  {
-    "bytes": "1",
-    "label": "_initializing",
-    "offset": 1,
-    "slot": "0",
-    "type": "bool"
-  },
-  {
-    "bytes": "30",
-    "label": "spacer_0_2_30",
-    "offset": 2,
-    "slot": "0",
-    "type": "bytes30"
```

Comment:
`_initialized/ing` replaced by expanding the spacer. No resulting changes to subsequent layout.